pkg/runtime/framework/plugins/mpi/mpi.go

```diff
@@ -175,7 +175,7 @@ func (m *MPI) EnforceMLPolicy(info *runtime.Info, trainJob *trainer.TrainJob) er
 						WithMountPath(*info.RuntimePolicy.MLPolicySource.MPI.SSHAuthMountPath),
 				}...,
 			)
-			if ps.Name == constants.JobLauncher && container.Name == constants.ContainerLauncher {
+			if ps.Name == constants.JobLauncher && container.Name == constants.Node {
```

Comment:
```suggestion
			if ps.Name == constants.JobLauncher && (container.Name == constants.Node || container.Name == constants.ContainerLauncher){
```

This should be considered for `runLauncherAsNode: false`
Ideally, we want to consider whether or not runLauncherAsNode here, but for now, we can just consider Launcher container